build.gradle.kts

```diff
@@ -79,6 +79,8 @@ tasks {
   withType<KotlinCompile> {
     dependsOn("buildLocationsInsidePrisonApiModel", "buildActivitiesAppointmentsApiModel", "buildPrisonApiModel", "buildManageUsersApiModel")
     compilerOptions.jvmTarget = org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_21
+    compilerOptions.freeCompilerArgs.add("-Xannotation-default-target=param-property")
+    compilerOptions.freeCompilerArgs.add("-Xwarning-level=IDENTITY_SENSITIVE_OPERATIONS_WITH_VALUE_TYPE:disabled")
```

Comment:
Just curious.  What does this actually mean/do?

Reply:
When you say we need to amend those places, what is the actual change required then?

Reply:
https://youtrack.jetbrains.com/issue/KT-78546/False-positive-IDENTITYSENSITIVEOPERATIONSWITHVALUETYPE-when-comparing-LocalDate-with-equality-operator

Effectively, avoid using `==` for dates, times, datetimes,  and use `.equals` instead.

Reply:
But looks like if/when we do, IntelliJ might warn on it..